src/Lumper.Lib/BSP/Lumps/GameLumps/Sprp.cs

```diff
@@ -11,45 +11,46 @@ public class Sprp(BspFile parent) : ManagedLump<GameLumpType>(parent)
     public StaticPropLump StaticProps { get; set; } = null!;
 
     private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
```

Comment:
`29e68f11c53f77be3d50bfa47be0eb2891ca5c36` commit message AAAAAA

Reply:
allow it

Reply:
okay fiiiine